aptos-move/aptos-vm-environment/src/prod_configs.rs

```diff
@@ -103,6 +108,16 @@ pub fn aptos_prod_verifier_config(features: &Features) -> VerifierConfig {
         enable_enum_types,
         enable_resource_access_control,
         enable_function_values,
+        max_function_return_values: if enable_function_values {
+            Some(128)
+        } else {
+            None
+        },
+        max_type_depth: if enable_function_values {
+            Some(20)
```

Comment:
Where does the 20 come from? I thought we have something like 128 for recursion depth, so this seems a little low.

Reply:
From the similar dynamic check limit:
https://github.com/aptos-labs/aptos-core/blob/main/aptos-move/aptos-gas-schedule/src/gas_schedule/transaction.rs#L271